### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by onojk** [cwensley/pablodraw#49] [General]
Hi I am Jonathan Kendall onojk123, I am interested in this project.  Is there any way I can help?  I am installing a new OS today on a new NvMe drive, thought I would give a try building pablodraw this evening.  Tips suggestions?  I will be using Ubuntu for the OS.

**Comment by cwensley** [comment]
Tips: use VS Code and .NET 6 SDK.  There are tasks for building on all platforms.

**Comment by onojk** [comment]
I ran the linux binary (I think its called a binary) and I was able to run the program perfectly.  I enjoyed it very much, made my first text based piece of art on it.  :-)
